assets/test/spec/services/dataServiceSpec.js

```diff
@@ -52,15 +52,13 @@ describe("DataService", function(){
       // Namespaced subresource with params
       [{type:'pods/proxy', id:"mypod", namespace:"myns", myparam1:"myvalue"}, "http://localhost:8443/api/v1beta3/namespaces/myns/pods/mypod/proxy?myparam1=myvalue"],
     ];
-    
-    for (var i=0; i < tc.length; i++) {
-      it("should generate a correct URL for " + JSON.stringify(tc[i][0]), function(tc){
-      	return function() {
-          expect(DataService.url(tc[0])).toEqual(tc[1]);
-        };
-      }(tc[i]));
-    }
-    
+
+    _.each(tc, function(item) {
```

Comment:
I think it might have been related to it() lazily evaluating the function passed in... make sure the `item` object seen inside the function() is actually iterating through all the test cases